README.md

```diff
@@ -1,16 +1,171 @@
-# Title
+# Nested Data Structures: Printing and Coalescing Data
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+* Display the cells in an `Array` of `Array`s
+* Traverse `Array` of `Array`s to produce a single value
+* Traverse `Array` of `Array`s to produce a new nested data structure
 
 ## Introduction
 
-## SWBAT 1
+When we started this module, we mentioned that we often use nested data
+structures as a "base" from which to do data processing. Arrays of arrays is
+our first milestone in learning to work with nested data.
 
-## SWBAT 2
+We're going to pick out three specific types of processing here:
+
+* Displaying the nested structure
+* Transforming the nested structure into a new structure
+* Transforming the nested structure into a result
+
+In each of these sections, we're going to give you a bit of "reference code."
+Reference code is code that's a little generic and probably doesn't solve any
+_real_ problem, but which is kept intentionally ***very simple*** so that you
+can see how it might be adapted to your particular need. Some people call these
+"reference implementations."
+
+## Display the cells in an `Array` of `Array`s
+
```

Comment:
If we're going to format it as `Array` of `Array`s should we do this for every instance?

Reply:
I'm thinking it's not necessary to tick off `Array` in this phrase at all?